File: app/src/main/java/com/olehliskovych/placenote/ui/labels/LabelsActivity.java
package com.olehliskovych.placenote.ui.labels;

import android.content.Context;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.util.AttributeSet;
import android.view.View;

import com.olehliskovych.placenote.databinding.ActivityLabelsBinding;
import com.olehliskovych.placenote.ui.base.BaseActivity;

public class LabelsActivity extends BaseActivity {

    private ActivityLabelsBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    protected void onResume() {
        super.onResume();

    }

    @Override
    protected void setupUI() {
        setSupportActionBar(binding.toolbar);
    }

    private void navigateToLabelList() {

    }
}

File: app/src/main/java/com/olehliskovych/placenote/ui/labels/LabelsActivity.kt
package com.olehliskovych.placenote.ui.labels

import android.databinding.DataBindingUtil
import android.os.Bundle
import com.olehliskovych.placenote.R

import com.olehliskovych.placenote.databinding.ActivityLabelsBinding
import com.olehliskovych.placenote.databinding.ActivityMainBinding
import com.olehliskovych.placenote.ui.base.BaseActivity

class LabelsActivity : BaseActivity() {

    private lateinit var binding: ActivityLabelsBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.setContentView(this, R.layout.activity_labels)
    }

    override fun onResume() {
        super.onResume()

    }

    override fun setupUI() {
        setSupportActionBar(binding!!.toolbar)
    }

    private fun navigateToLabelList() {

    }
}
